content/de/verein/event/fireshonks-workshop-2022/index.md

```diff
@@ -0,0 +1,20 @@
+{
+    "title": "Datenschutzrechte zum Mitmachen – DSGVO-Anfragen stellen mit Datenanfragen.de",
+    "type": "event",
+    "event_date": "2022-12-29T15:30:00+02:00",
+    "duration": "90m",
+    "place": {
+        "online": "https://meeten.statt-drosseln.de/b/mer-uld-o2j-uvp"
```

Comment:
Is this public information?

Reply:
https://pretalx.c3voc.de/fire-shonks-2022/talk/TBBNKB/